lib/cli/cliDebug.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+const CLI = require('./messages.js');
+const CLICommand = require('./cliCommand.js');
+const tools = require('../tools.js');
+const fs = require('fs');
+const path = require('path');
+const child_process = require('child_process');
+
+/** Command ioBroker compact ... */
+module.exports = class CLICompact extends CLICommand {
+    /** @param {import('./cliCommand.js').CLICommandOptions} options */
+    constructor(options) {
+        super(options);
+    }
+
+    /**
+     * Executes a command
+     * @param {any[]} args
+     */
+    execute(args) {
+        const { callback, ...params } = this.options;
+        const adapter = args[0];
+        if (!adapter) {
+            CLI.error.requiredArgumentMissing(
+                'adaptername',
+                'debug <adaptername>'
+            );
+            return void callback(34);
+        }
+
+        const adapterDir = tools.getAdapterDir(adapter);
+        if (!adapterDir) {
+            CLI.error.adapterDirNotFound(adapter);
+            return void callback(34);
+        }
+
+        let ioPack;
+        try {
+            ioPack = JSON.parse(
+                fs.readFileSync(
+                    path.join(adapterDir, 'io-package.json'),
+                    'utf8'
+                )
+            );
+        } catch (e) {
+            CLI.error.cannotLoadIoPackage(adapter);
+            return void callback(34);
+        }
+
+        const possibleMainFiles = [
```

Comment:
package.json main attribute should also work

Reply:
added.